platform/broadcom/sonic-platform-modules-ruijie/common/lib/eepromutil/fantlv.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+class FantlvException(Exception):
+    def __init__(self,  message='fantlverror', code=-100):
+        err = 'errcode: {0} message:{1}'.format(code, message)
+        Exception.__init__(self, err)
+        self.code = code
+        self.message = message
+
+class fan_tlv(object):
+    HEAD_INFO = "\x01\x7e\x01\xf1"
+    VERSION = 0x01                       # E2PROM文件定义的版本号，初始版本为0x01
```

Comment:
this makes difficult for english only speaker to review. can these comments be written in english?

Reply:
done!